CovertInfiltration/Config/XComGameData.ini

```diff
@@ -266,6 +266,32 @@ bNegativeEffect=true
 +TacticalGameplayTags=SITREP_OpportuneMoment2
 +PositiveEffects=OpportuneMoment2Effect
 
+[MessyInsertion X2SitRepTemplate]
++TacticalGameplayTags=SITREP_MessyInsertion
++NegativeEffects=MessyInsertionEffect
++bNegativeEffect=true
++MinimumForceLevel=6
+
+[RestrictedAirspace X2SitRepTemplate]
++TacticalGameplayTags=SITREP_RestrictedAirspace
++bNegativeEffect=true
++MinimumForceLevel=6
++ValidMissionFamilies="Recover"
++ValidMissionFamilies="Hack"
++ValidMissionFamilies="DestroyObject"
++ValidMissionFamilies="NeutralizeFieldCommander"
++ValidMissionFamilies="SabotageTransmitter"
```

Comment:
Could you explain the idea behind this?

Reply:
It doesn't make sense in retaliations, it's irrelevant in fixed-evac missions, and it pretty much wrecks the game if it appears on a supply extract.

Reply:
cc #484